### ISSUE TITLE
Slow preview on some WP Network installs

### ISSUE THREAD

**Issue by mleroi** [uncatcrea/wp-appkit#166]
It seems that app preview is very slow (like 1 minute to load the app) on some WP Network installs (seems to depend on the hosting service used).

**Comment by lpointet** [comment]
@mleroi It may be related to the readfile() trick we just discussed at #wcparis, no?

**Comment by mleroi** [comment]
@lpointet yes that's what I'm thinking. Did you notice slow preview when testing Network on your host? (me I experienced that on one on my shared hosting).

**Comment by lpointet** [comment]
@mleroi Nope, preview perf is obviously the same on my network as the one on a single install, with the same hosting platform.
